src/screens/regimen_setup/styles.ts

```diff
@@ -113,3 +113,11 @@ export const ButtonContainer = styled.View`
   width: ${deviceWidth * 0.8}px;
   justify-content: space-between;
 `;
+
+export const AnswerOptionSelected = styled.TouchableOpacity`
+  width: ${deviceWidth * 0.8}px;
```

Comment:
this can be changed and better-using percentages like 100% for full-screen width or 90%